src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/PartyFlagsMapping.java

```diff
@@ -137,7 +137,7 @@ public static PartyFlags urgentCase(SscsCaseData caseData) {
 
     public static PartyFlags adjournCaseInterpreterLanguage(SscsCaseData caseData) {
         PartyFlags adjournCasePartyFlag = null;
-        if (isNotBlank(caseData.getAdjournCaseInterpreterLanguage())) {
+        if (caseData.getAdjournCaseInterpreterLanguage() != null && isNotBlank(caseData.getAdjournCaseInterpreterLanguage().getValue().getCode())) {
```

Comment:
I am not sure if you should be overriding this value for the Manual Hearing Route through ExUi, can you confirm with Makeda?